addusers.py

```diff
@@ -46,7 +46,7 @@
 from moc_utils import get_absolute_path, select_rows
 from quotas import QuotaManager
 from setpass import SetpassClient, random_password
-from config import set_config_file
+from config import set_config_file, user_column
```

Comment:
Instead of importing this from config.py, it should be read in from the settings.ini file using ConfigParser.  If you look at line 354-358 where the admin credentials are loaded, you can see how to do this.

For now, you can assume the `config` variable from `__main__` will be inherited in the functions.  There are some other functions that also do this at the moment, it will be changed later when the configuration loading is streamlined.